vendor/github.com/coreos/etcd/cmd/vendor/github.com/coreos/pkg/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
Now, we need keep the complie success. I has create jira(JAV-44) to  remove 3rd software code.